tests/upgrade_v2_script/static/fluent_bit.output.yaml

```diff
@@ -36,7 +36,9 @@ fluent-bit:
     - effect: NoSchedule
       operator: Exists
   image:
-    pullPolicy: IfNotPresent
+    repository: public.ecr.aws/sumologic/fluent-bit
+    tag: 1.6.0
```

Comment:
should be `1.6.10`

Reply:
This should be whatever is in the input file (which is `1.6.0`)

Reply:
what if it is same value like in our old `values.yaml`? IMHO we should migrate it

Reply:
and 1.6.0 is corrupted

Reply:
I don't agree, we should not change the values. Let me add info about new default value though.

Reply:
waiting.. :D

Reply:
Done :)